### PR TITLE
Set up repository for new component

### PR DESCRIPTION
**Pull request by andrecardoso**
- Adapt structure to follow the new Rise components architecture
- Change Circle CI configuration
- Add simple rise-slides implementation

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) Create rise-slides repository on github](https://trello.com/c/1LkHfOse/62-1-create-rise-slides-repository-on-github)

**Comment by andrecardoso** [comment]
[(1) Add simple skeleton component files](https://trello.com/c/fHFujrMB/61-1-add-simple-skeleton-component-files)

**Comment by andrecardoso** [comment]
[(1) Set up build and deployment (staging and production) CI](https://trello.com/c/jYPQ855e/63-1-set-up-build-and-deployment-staging-and-production-ci)

File: src/slides.js [hunk 1]
```diff
@@ -19,48 +16,49 @@ export default class Slides {
...
-    frame.style.visibility = 'hidden';
-    frame.style.position = 'absolute';
-    frame.setAttribute('frameborder', '0');
-    frame.setAttribute('allowTransparency', 'true');
-    frame.setAttribute('allowfullscreen', 'true');
-    frame.setAttribute('mozallowfullscreen', 'true');
-    frame.setAttribute('webkitallowfullscreen', 'true');
+    frame.className = "webpage-frame";
+    frame.style.visibility = "hidden";
+    frame.style.position = "absolute";
```
Comment (olegrise) [inline src/slides.js]: @andrecardoso I'm not sure we need to assign `frame.className` and `frame.style.position`, but it's okay to leave them for this PR.

Comment (andrecardoso) [reply src/slides.js]: Yeah.. that's what already exist, I'm just applying double quotes to match the lint rules.

**Pull request merged**